jest.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+    preset: 'ts-jest/presets/js-with-ts',
+    testEnvironment: 'node',
+    transformIgnorePatterns: ['/node_modules/(?!antlr4/)'],
```

Comment:
I assume you just copied this from chainner, so this is also a comment for it as well: since Navi is in its own repo, do we need antlr here?

Reply:
No, we don't. Good point.